### PR TITLE
Fix misleading UI numbers

### LINKED ISSUE #2006: num docs as displayed in the UI is misleading
As reported in #1987.

The UI reports the overall number of docs in splits. (including splits marked as deleted).

https://github.com/quickwit-oss/quickwit/blob/f32f31694b0815da2b0eb63f70db122a23421bee/quickwit/quickwit-ui/src/components/IndexSummary.tsx#L43-L49

This is quite misleading for users.

We probably want 

- num docs (defined as published splits only)
- num published splits

Size is fishy too. I don't know how it is computed. We need to edit the lable to tell if this is the split size of the raw document size, and make sure this is just published stuff if it is the latter.

And possibly:
- num staged splits
- num splits marked as deleted

### PR DESCRIPTION
**Pull request by boraarslan**
Fixes #2006 

I also changed the style a little bit. (Thanks @Fursum)

It currently looks like this:
image

### CODE REVIEW

**Review by fmassot — APPROVED** [review]
LGTM! Nice job!

File: quickwit/quickwit-ui/src/components/IndexSummary.tsx [hunk 1]
```diff
@@ -34,57 +35,69 @@ const Row = styled.div`
...
+        <IndexRow title="Size of published splits:">
+          <NumberFormat
+            value={total_num_bytes / 1000000}
+            displayType={"text"}
+            thousandSeparator={true}
+            suffix=" MB"
+            decimalScale={2}
+          />
+        </IndexRow>
+        <IndexRow title="Number of documents (Published):">
```
Comment (guilload) [inline quickwit/quickwit-ui/src/components/IndexSummary.tsx:91]:
```suggestion
        <IndexRow title="Number of published documents:">
```

Comment (guilload) [reply quickwit/quickwit-ui/src/components/IndexSummary.tsx:91]: ?

File: quickwit/quickwit-ui/src/utils/models.ts [hunk 2]
```diff
@@ -253,4 +252,4 @@ export type NodeId = {
   id: string,
   grpc_address: string,
   self: boolean,
-}
```
Comment (guilload) [inline quickwit/quickwit-ui/src/utils/models.ts:256]: :D

Comment (boraarslan) [reply quickwit/quickwit-ui/src/utils/models.ts:256]: LOL

**Pull request merged**